packages/flutter_timeline_view/lib/src/config/timeline_options.dart

```diff
@@ -141,6 +142,9 @@ class TimelineOptions {
 
   /// Maximum length of the post content
   final int? maxContentLength;
+
+  /// Boolean to enable timeline post overview screen before submitting
+  final bool enablePostOverviewScreen;
```

Comment:
this setting should be moved to https://github.com/Iconica-Development/flutter_timeline/blob/master/packages/flutter_timeline/lib/src/models/timeline_configuration.dart
And be implemented in the timeline userstory